projects/aws/bottlerocket-bootstrap/Makefile

```diff
@@ -13,7 +13,7 @@ SOURCE_PATTERNS=.
 REPO_SUBPATH=..
 
 GIT_TAG=v$(RELEASE_BRANCH)-$(shell yq e ".$(RELEASE_BRANCH).number" $(BASE_DIRECTORY)/EKSD_LATEST_RELEASES)
-
+LATEST_TAG=$(GIT_TAG)-$(LATEST)
```

Comment:
This should already be set from the common.mk... I'll take a closer look here

Reply:
Yeah remove these changes and instead move https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L35 to  https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L100